packages/frontend/src/components/Game/GameBoard/ResultDisplay/index.jsx

```diff
@@ -40,20 +42,24 @@ const ResultDisplay = () => {
             <h2 className="lh-lg">Its a tie!</h2>
           ) : (
             <h2 className={`text-${winner.color.className}`}>
-              {`${winner.name} wins!`}
+              {winner.name === 'Player 1'
+                ? `${formValues.player1Name}`
+                : `${formValues.player2Name}`}{' '}
+              wins!
```

Comment:
I'm realizing that the players entered name form the form never gets entered in the game context. I'n the game context its always Player 1 and Player 2 under name in that context, thats why I wrote the code like this and then getting the entered name out of the form context like this. 

Do you think we need it so that the entered name from the form is also part of the game context?

Reply:
I think so. The way I see it, the player's names should be stored in the player objects in the game context and used wherever the player names are rendered (Scoreboard, game over screen, etc.)

Reply:
@perennialAutodidact Totally, I just wasn't sure if we needed the information in more than one context. On one hand there is a whole new step to saving the players info in both contexts, the form and the game context. It almost makes the form context not necessary.

Reply:
The form context is just for collecting the form data. It might be a little unnecessary, but I always vie on the side of more organization of data over one monolithic state object. I'd rather shift data around as needed and have only the data that's needed for the various parts of the app. It probably saves on memory too because the state object is copied every time a state value changes. 

The user has the opportunity to go back and adjust it and then once they submit the form, the relevant info goes in the game context and the form context should be cleared. Without the form context, there isn't a place to keep things like the image search query. The form might have other values later that might not apply to the game context, so they'd go in the form context until they were used.